CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Imperative package will be documented in this file.
 
 ## Recent Changes
 
+- Remove temporary build modifications that were used to enable zowe-cli to build with an unpublished imperative.
 - Fix tests for interactive prompting.
 - Add the --dd flag to profile creation to allow the profile to be created without the default values specified for that profile.
 - Use a token for authentication if a token is present in the underlying REST session object.
```

Comment:
Should the line below in the changelog (line 11) be updated, to mention "ConnectionPropsForSessCfg" instead of "CredsForSessCfg"?

Reply:
Done.